README.md

```diff
@@ -1,149 +1,161 @@
 
--   This action runs using Node 20. If you are using self-hosted GitHub Actions
-    runners, you must use a [runner
-    version](https://github.com/actions/virtual-environments) that supports this
-    version or later.
+* This action requires a Google Cloud service account which has the **Security Posture Shift-Left Validator** role or the **Security Posture Admin** role on the Google Cloud organization that includes the IaC resources. For more information, see [Authorization](#authorization).
+
+
+* This action runs using Node 20. If you are using self-hosted GitHub Actions
+  runners, you must use a [runner
+  version](https://github.com/actions/virtual-environments) that supports this
+  version or later.
+
 
 ## Usage
 
+
 ```yaml
 jobs:
-  job_id:
-    permissions:
-      contents: 'read'
-      id-token: 'write'
-
-    steps:
-    - uses: 'actions/checkout@v4'
-    - id: 'auth'
-      uses: 'google-github-actions/auth@v2'
-      with:
-        workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
-        service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
-
-    - id: 'analyze-code-security-scc'
-      uses: 'google-github-actions/analyze-code-security-scc@v1'
-      with:
-        organization_id: '123456789'
-        scan_file_ref: './tf_plan.json'
-        iac_type: 'terraform'
-        iac_version: '1.0.0'
-        scan_timeout: '1m'
-        ignore_violations: false
-        failure_criteria: 'High:1,Medium:1,Low:1,Operator:OR'
-        fail_silently: true
-
-    - if: ${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != '' }}
-        uses: 'actions/upload-artifact@v4'
-        with:
-          name: 'sarif'
-          path: '${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}'
+ job_id:
+   permissions:
+     contents: 'read'
+     id-token: 'write'
+
+
+   steps:
+   - uses: 'actions/checkout@v4' 
+   - id: 'auth'
+     uses: 'google-github-actions/auth@v2'
+     with:
+       workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+       service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
+
+
+   - id: 'analyze-code-security-scc'
+     uses: 'google-github-actions/analyze-code-security-scc@v1'
+     with:
+       organization_id: '123456789'
+       scan_file_ref: './tf_plan.json'
+       iac_type: terraform
+       iac_version: '1.0.0'
+       scan_timeout: '1m'
+       ignore_violations: false
+       failure_criteria: 'High:1,Medium:1,Low:1,Operator:or'
+       fail_silently: false
+
+
+   - if: ${{steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''}}
+       uses: actions/upload-artifact@v4
+       with:
+         name: sarif
+         path: ${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}
```

Comment:
Make sure all non-boolean yaml attributes are wrapped in single quotes (throughout)

```suggestion
         path: '${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}'
```

Reply:
Done.